app/models/edition.rb

```diff
@@ -33,6 +33,8 @@ class ResurrectionError < RuntimeError
   field :change_note,          type: String
   field :review_requested_at,  type: DateTime
 
+  field :auth_bypass_id,       type: String, default: -> { SecureRandom.uuid }
```

Comment:
It looks like this default is made superfluous by the subsequent commit that introduces a callback. Might be better to add this default in at the point when it will be impactful.

Reply:
Hmm, true... I'll remove that from this PR.

Reply:
Removed this in https://github.com/alphagov/publisher/pull/1313/commits/e9b66c1a81d6e64a5aab0219fe2c120a22a01a9c